packages/shared/src/components/fields/Dropdown.tsx

```diff
@@ -124,7 +126,7 @@ export function Dropdown({
       <Menu
         disableBoundariesCheck
         id={id}
-        className={`menu-primary ${scrollable && 'scrollable'}`}
+        className={`${scrollable && 'scrollable'} menu-${menuType}`}
```

Comment:
String interpolation is not good for Tailwind JIT.
I think that instead of providing the `menuType` prop we can just provide the right class outside of this component.
Exactly like we do with buttons.